internal/resources/providers/azurelib/inventory/susbcriptions_provider.go

```diff
@@ -54,14 +66,37 @@ func NewSubscriptionProvider(log *logp.Logger, credentials azcore.TokenCredentia
 		},
 	}
 
+	tenantClient := tenantAzureClientWrapper{
+		Tenants: func(ctx context.Context, clientOptions *arm.ClientOptions, options *armsubscriptions.TenantsClientListOptions) ([]armsubscriptions.TenantsClientListResponse, error) {
+			c, err := armsubscriptions.NewTenantsClient(credentials, clientOptions)
```

Comment:
Can't we get tenants from the graph api?

Reply:
I checked and I haven't found a way to do it. If you know a way, it would be cool.
Here's the ticket for [the resources we can pull using Resource Graph](https://github.com/elastic/security-team/issues/10160). Everything else has its own separate tickets.

Reply:
I do see a resource called `microsoft.aadiam/tenants`, but using the graph explorer in my count leads to 0 results. Not sure if it's because this resource is about something else or permission issues. Either way I hate how little documentation and context azure has.

So let's go with your call!